CHANGELOG.md

```diff
@@ -1,3 +1,15 @@
+## 0.11.1 (2018-01-08)
+
+A small release fixing a somewhat nasty bug involving running benchmarks in
+parallel.
+
+### Bugfixes (User Facing)
+
+* If you were running benchmarks in parallel, you would see results for each
+parallel process you were running. So, if you were running two jobs, and
+setting your configuration to `parallel: 2`, you would see four results in the
+formatter. This is now correctly showing only the two jobs.
```

Comment:
Imo we could go **four** and **two** here (I sometimes like highlights) :)

Reply:
Done! 👍